tools/ldb_cmd.cc

```diff
@@ -3041,8 +3120,8 @@ void MultiGetEntityCommand::DoCommand() {
     key_slices.emplace_back(key);
   }
 
-  db_->MultiGetEntity(ReadOptions(), GetCfHandle(), num_keys, key_slices.data(),
-                      results.data(), statuses.data());
+  db_->MultiGetEntity(ropt, cfh, num_keys, key_slices.data(), results.data(),
+                      statuses.data());
```

Comment:
We also have the `scan` command for range scans; we might want to support the new parameter there as well.

Reply:
Somewhat related but would it make sense to similarly teach the various "dump" commands (e.g. dump_wal, idump, ...) about timestamps? (If yes, we can do it as a separate follow-up.)

Reply:
Thanks for catching the missing scan support. Yes, that's a good call! Augmenting "dump" commands helps with inspecting the DB and debug. I will work on this in a follow up.